docs/sources/Cisco/index.md

```diff
@@ -608,4 +608,52 @@ Use the following search to validate events are present
 index=email sourcetype=cisco:esa:*
 ```
 
-Verify timestamp, and host values match as expected
\ No newline at end of file
+Verify timestamp, and host values match as expected
+
+## Product - UCS-Hyperflex
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on  | na                                                               |
+| Product Manual | multiple |
+
+
+### Sourcetypes
+
+| sourcetype     | notes                                                                                                   |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| cisco:ucs_hx    |  None                                                                                                    |
+
+### Sourcetype and Index Configuration
+
+| key            | sourcetype     | index          | notes          |
+|----------------|----------------|----------------|----------------|
+| cisco_ucs_hx    | cisco:ucs_hx    | main          | None     |
```

Comment:
unless there is an existing TA using _ in the sourcetype we should use cisco:ucs:hx

Reply:
Made the Changes